docs/install_guides/installing-model-dev-tools.md

```diff
@@ -14,12 +14,12 @@ In both cases, Python 3.6 - 3.9 need be installed on your machine before startin
 
 > **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. 
 
-## <a name="python-developers"></a>For Python Developers
-
-If you are a Python developer, follow the steps in the <a href="openvino_docs_install_guides_install_dev_tools.html#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
-   
+<a name="python-developers"></a>
+## For Python Developers
+If you are a Python developer, you can find the main steps below to install OpenVINO Development Tools. For more details, see <https://pypi.org/project/openvino-dev>.
```

Comment:
This paragraph is out-of-date. This article has been updated by another PR for 2022.2. Also we can change the link `<a href="openvino_docs_install_guides_install_dev_tools.html#install-dev-tools">` to `<a href="#install-dev-tools">` here.